tools.json

```diff
@@ -669,5 +669,13 @@
     "description": "A Terminal Client for Redis with AutoCompletion and Syntax Highlighting.",
     "url": "https://iredis.io",
     "authors": ["laixintao"]
+  },
+
+  {
+    "name": "flask-redisboard",
+    "language": "Python",
+    "repository": "https://github.com/hjlarry/flask-redisboard",
+    "description": "A flask extension to support user view and manage redis with beautiful interface.",
+    "authors": ["hjlarry"]
```

Comment:
Hi @hjlarry - the 'authors' should contain a Twitter, not GitHub, handle. Please confirm that you are https://twitter.com/hjlarry or simply leave the array empty if not. Thanks :)

Reply:
yep, https://twitter.com/hjlarry  is my twitter. Thank you